.travis.yml

```diff
@@ -74,7 +74,7 @@ script:
   - ./typo3cms configuration:showLocal BE/installToolPassword
   - ./typo3cms configuration:showActive BE/installToolPassword
   - ./typo3cms configuration:set BE/installToolPassword foobar && grep installToolPassword $PATH_WEB/typo3conf/LocalConfiguration.php | grep foobar
-  - ./typo3cms configuration:remove BE/installToolPassword --force && [ -z "$(grep installToolPassword $PATH_WEB/typo3conf/LocalConfiguration.php)" ]
+  - ./typo3cms configuration:remove BE/installToolPassword --force && grep -v installToolPassword $PATH_WEB/typo3conf/LocalConfiguration.php
```

Comment:
Hm, indeed the return value is correct with `-v`. Can you please make it `-qv` so that we suppress the output? Thanks